src/app/Scenes/Artwork/Components/ArtworkScreenHeader.tsx

```diff
@@ -117,19 +104,16 @@ const ArtworkScreenHeader: React.FC<ArtworkScreenHeaderProps> = ({ artwork }) =>
       </Flex>
 
       <Flex flexDirection="row" alignItems="center">
-        <Button
-          icon={<SaveIcon isSaved={!!isSaved} />}
-          size="small"
-          variant="text"
+        <Touchable
           accessibilityRole="button"
-          accessibilityLabel={saveButtonText()}
+          accessibilityLabel="Save button"
```

Comment:
super minor but can we change the a11y label to:
```suggestion
          accessibilityLabel="Save artwork"
```

Reply:
Done